packages/tests/src/components/bd-checkbox.ts

```diff
@@ -0,0 +1,16 @@
+import BigDesignElement from './bd-element';
+
+// TODO: Make more typescripty
+export default class BigDesignCheckbox extends BigDesignElement {
+  toggle = (): void => {
+    this.getBoxElement().click()
+    // tslint:disable-next-line:no-console
+    console.log(`Toggled checkbox with selector: ${this.selector}`)
+  }
+
+  private getBoxElement = (): WebdriverIO.Element => {
```

Comment:
same here, `private getBoxElement() { ... }` should be enough from what I can tell

Reply:
✅